core/libp2p/connection/raw_connection.hpp

```diff
@@ -7,15 +7,28 @@
 #define KAGOME_RAW_CONNECTION_HPP
 
 #include "libp2p/basic/readwritecloser.hpp"
+#include "libp2p/event/emitter.hpp"
+#include "libp2p/multi/multiaddress.hpp"
 
 namespace libp2p::connection {
 
-  struct RawConnection : public basic::ReadWriteCloser {
+  struct RawConnection
```

Comment:
Please, add a doc, that it's the "lowest" connection possible, never for actual usage, or something like that